docs/tutorials/notebooks/basic_ot_between_datasets.ipynb

```diff
@@ -0,0 +1,387 @@
+{
```

Comment:
<p>"that is a way to measure distances between points" -&gt; "that is a way to measure a ground cost between points"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>